gulpfile.babel.js

```diff
@@ -116,12 +116,20 @@ function fontCssPipe() {
 
 function bmp2svg() {
   return through.obj(function(file, enc, next) {
-
     var potraceProcess = child_process.spawn(
       'potrace', ['-o', '-', '-b', 'svg'], {
          stdio: [ 'pipe', 'pipe', 'inherit' ]
        }
     );
+    potraceProcess.on("error", () => {
```

Comment:
If I'm reading this correctly, we'll get this message for any error when invoking potrace. Is it possible to handle ENOENT specifically here?